### DISCUSSION TITLE
Spa auto-off after 72 hours defeated!

### DISCUSSION THREAD

**Discussion by WesWesthaver** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#619] [General]
I purchased a SaluSpa Paris one year ago and tried to leave the spa running all the time so it would be ready any time I wanted to take a dip. I discovered that if I didn't tickle the pump buttons at least every other day the pump would shut off. The spa paperwork says this is to be expected after 72 hours of inactivity. However, the paperwork offers two different ways of waking up the pump after it is shut down:

[Paperwork #1](http://www.westhaver.com/SaluSpa_1.jpg) 
[Paperwork #2](http://www.westhaver.com/SaluSpa_2.jpg) 

So, I installed the Lay-Z-Spa Module to see if the auto-off function could be avoided. I'm happy to report that it can. I set up a series of commands in the command queue:

```
// Wake the spa up at 11:30 AM
Set brightness:8, 11:30:00 AM, Interval: 86400s
Set target:100,   11:30:05 AM, Interval: 86400s
Set heat:1,       11:30:10 AM, Interval: 86400s

// Put the spa to sleep at 11:30 PM
Set pump:0,       11:30:00 PM, Interval: 86400s
Set brightness:0, 11:30:05 PM, Interval: 86400s
```
This has provided a couple of benefits: 1) the spa doesn't shut down automatically any more, and 2) the pump is only running half as much so hopefully it will have a longer life.

-Wes

PS. @visualapproach I sent you some beer money!

**Comment by visualapproach** [comment]
@WesWesthaver Thank you sir, and cheers! Happy to hear that you're satisfied with it!
